packages/openactive-broker-microservice/app.js

```diff
@@ -1083,14 +1083,76 @@ async function ingestParentOpportunityPage(rpdePage, feedIdentifier, validateIte
 
   for (const item of items) {
     const feedItemIdentifier = feedPrefix + item.id;
-    if (item.state === 'deleted') {
-      const jsonLdId = state.parentOpportunityRpdeMap.get(feedItemIdentifier);
-      state.parentOpportunityMap.delete(jsonLdId);
-      state.parentOpportunityRpdeMap.delete(feedItemIdentifier);
-    } else {
+
+    if (item.state !== 'deleted') {
       const jsonLdId = item.data['@id'] || item.data.id;
+
       state.parentOpportunityRpdeMap.set(feedItemIdentifier, jsonLdId);
       state.parentOpportunityMap.set(jsonLdId, item.data);
+
+      // If there are subEvents then we have a basic "small provider" SessionSeries feed. This is not
+      // recommended, but we support it anyway here. We do this by converting each of the embedded
+      // subEvents to its own opportunityItem, taking information from the subEvent itself as well as
+      // from the item in which it is embedded. This mimics the way that the event would be seen in a
+      // ScheduledSession feed.
+      if (item.data.subEvent) {
+        for (const subEvent of item.data.subEvent) {
+          const opportunityItemData = {
+            ...subEvent,
+          };
+          opportunityItemData['@context'] = item.data['@context'] || item.data.context;
```

Comment:
Don't need this as `@context` will always include an `@`
```suggestion
          opportunityItemData['@context'] = item.data['@context'];
```